src/patternfly/components/FormControl/form-control.scss

```diff
@@ -83,161 +56,110 @@ $pf-v5-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 2
-  --#{$form-control}--success--BackgroundPositionX: calc(100% - var(--#{$form-control}--PaddingLeft));
-  --#{$form-control}--success--BackgroundPositionY: center;
-  --#{$form-control}--success--BackgroundPosition: var(--#{$form-control}--success--BackgroundPositionX) var(--#{$form-control}--success--BackgroundPositionY);
-  --#{$form-control}--success--BackgroundSizeX: var(--#{$form-control}--FontSize);
-  --#{$form-control}--success--BackgroundSizeY: var(--#{$form-control}--FontSize);
-  --#{$form-control}--success--BackgroundSize: var(--#{$form-control}--success--BackgroundSizeX) var(--#{$form-control}--success--BackgroundSizeY);
-  --#{$form-control}--success--BackgroundUrl: #{pf-bg-svg($pf-v5-c-form-control--success--Coordinates, $svg-color: $pf-v5-c-form-control--success--Color)};
+  --#{$form-control}--m-success--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--md);
+  --#{$form-control}--m-success--PaddingBottom: calc(var(--#{$pf-global}--spacer--form-element) - var(--#{$form-control}--m-success--BorderBottomWidth));
+  --#{$form-control}--m-success--BorderBottomColor: var(--#{$pf-global}--success-color--100);
+  --#{$form-control}--m-success--PaddingRight: var(--#{$pf-global}--spacer--xl);
 
   // Input m-warning
   --#{$form-control}--m-warning--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--md);
   --#{$form-control}--m-warning--PaddingBottom: calc(var(--#{$pf-global}--spacer--form-element) - var(--#{$form-control}--m-warning--BorderBottomWidth));
   --#{$form-control}--m-warning--BorderBottomColor: var(--#{$pf-global}--warning-color--100);
   --#{$form-control}--m-warning--PaddingRight: var(--#{$pf-global}--spacer--xl);
-  --#{$form-control}--m-warning--BackgroundPositionX: calc(100% - calc(var(--#{$form-control}--PaddingLeft) - #{pf-size-prem(1px)}));
-  --#{$form-control}--m-warning--BackgroundPositionY: center;
-  --#{$form-control}--m-warning--BackgroundPosition: var(--#{$form-control}--m-warning--BackgroundPositionX) var(--#{$form-control}--m-warning--BackgroundPositionY);
-  --#{$form-control}--m-warning--BackgroundSizeX: #{pf-size-prem(20px)};
-  --#{$form-control}--m-warning--BackgroundSizeY: var(--#{$form-control}--FontSize);
-  --#{$form-control}--m-warning--BackgroundSize: var(--#{$form-control}--m-warning--BackgroundSizeX) var(--#{$form-control}--m-warning--BackgroundSizeY);
-  --#{$form-control}--m-warning--BackgroundUrl: #{pf-bg-svg($pf-v5-c-form-control--m-warning--Coordinates, $svg-color: $pf-v5-c-form-control--m-warning--Color)};
 
   // Input invalid
   --#{$form-control}--invalid--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--md);
   --#{$form-control}--invalid--PaddingBottom: calc(var(--#{$pf-global}--spacer--form-element) - var(--#{$form-control}--invalid--BorderBottomWidth));
   --#{$form-control}--invalid--BorderBottomColor: var(--#{$pf-global}--danger-color--100);
   --#{$form-control}--invalid--PaddingRight: var(--#{$pf-global}--spacer--xl);
```

Comment:
Not a blocker for this PR, but @mcoker would it make sense to update these `--invalid` vars to `...--m-invalid`?

Reply:
@thatblindgeye yep, nice catch. Since it's a modifier now, this would be `--m-invalid`

Reply:
I've been debating on this. It could also be `--m-error`